releases.md

```diff
@@ -1,3 +1,36 @@
+# Release 21st of May 2024
+
+- Add functionality to bin price charts on the DEX into time windows such as 1h, 4h, 1d etc
+- Improve error messages shown in the Staking platform
+
+# Release 13th of May 2024
+
+- Significant Staking platform refactoring and performance improvements. Loading staking vaults is now more than twice as fast.
+
+# Release 7th of May 2024
+
+- Add $NVL token
+- Update $MILK token to use the correct muesli icon
+
+# Release 6th of May 2024
+
+- Add reward distribution differentiation between Accelerated Yield and Vault Staking rewards
+- Staking V2 goes live with changes including GENS flex staking as well as Rewards Multiplying revenue sharing staking vaults
+
+# Release 1st of May 2024
+
+- Staking V2 maintenance
+- Staking V2 UI changes
+
+# Release 22nd of April 2024
+
+- Candlestick option is added to the DEX price chart
+- Add support for multiple Plutus smart contract scripts
```

Comment:
Is this a feature we should include here? This seems internal and unrelated to what users could experience or have context to. These release notes should be related to features that users can experience not a summary of the work we do.